victor/src/layout/boxes/construct.rs

```diff
@@ -169,71 +169,82 @@ impl<'a> BlockContainerBuilder<'a> {
         )
     }
 
-    fn handle_text(&mut self, descendant: dom::NodeId, contents: &str) -> Option<dom::NodeId> {
-        if let Some(text) = self.make_text_run(contents) {
+    fn handle_text(&mut self, descendant: dom::NodeId, input: &str) -> Option<dom::NodeId> {
```

Comment:
Doesn't that make [this comment](https://github.com/SimonSapin/victor/blob/ce57a79746f30068b14821159e939dcd24febdab/victor/src/layout/boxes/mod.rs#L72-L73) obsolete?

Reply:
Yes.